examples/acis/Mapping_Example.py

```diff
@@ -0,0 +1,147 @@
+# September 12, 2013. We know we are interested in precipitation totals,
+# but we are also going to take advantage of the long-term record in ACIS
+# and ask it to return what the departure from normal precipitation was on
+# this day.
+
+parameters = {"state":"co","sdate":"20130909","edate":"20130912","elems":[
+    {"name":"pcpn","interval":"dly"},
+    {"name":"pcpn","interval":"dly","normal":"departure"}]}
+
+method = "MultiStnData"
+###########################################
+# In this case, rather than using station ID's, we are able to specify a new
+# parameter called 'state'. If we were interested in other states, we could just
+# add another to the list like this: "co,wy". Also notice how we are getting
+# both the precipitation and departure from normal within one variable. We'll
+# see how this changes the final data dictionary. Now let's make our call and
+# review our data.
+
+myData = acis_request(method,parameters)
+
+print(myData)
+
+###########################################
+# MultiStnData calls take longer to return than single stations, especially when
+# you request multiple states. We can see the data is divided by station, with
+# each station having it's own meta and data components. This time we also have
+# multiple values in each data list. Each value corresponds to the variable we
+# requested, in the order we requested it. So in this case, we have the
+# precipitation value, followed by the departure from normal value. Before we
+# plot this information, we need to add up the precipitation sums. But rather
+# than doing it in Python, let's make another ACIS call that prepares this for
+# us.
+
+parameters = {"state":"co","sdate":"20130909","edate":"20130912","elems":[
+    {"name":"pcpn","interval":"dly","smry":"sum","smry_only":1},
+    {"name":"pcpn","interval":"dly","smry":"sum","smry_only":1,"normal":"departure"}]}
+
+myData = acis_request(method,parameters)
+
+print(myData)
+
+###########################################
+# First of all, we have two new components to our elements: 'smry' and 'smry_only'.
+# 'smry' allows us to summarize the data over our time period. There are a few
+# options for this, including being able to count the number of records exceeding
+# a threshold (something we will explore in the next example). The other parameter,
+# 'smry_only', allows us to only return the summary value and not the intermediate
+# data.
+#
+# Now let's look at how our data has changed. Rather than having a just a 'meta'
+# and 'data' component, we have a new one called 'smry'. As you've guessed,
+# this contains our summary information (also in the order we requested it).
+# By specifying 'smry_only', there is no 'data' component. If we also wanted
+# all 4 days of data, we would simply remove that parameter.
+#
+# To wrap up this example, we will finally plot our precipitation sums and
+# departures onto a map using CartoPy and MetPy. To do this we will utilize
```

Comment:
It doesn't look like you're actually using MetPy here.

Reply:
You caught me! I was, but then I noticed the implementation I had was poor. Looks like I forgot to proofread that part of the tutorial afterwards.